lib/stat_tracker.rb

```diff
@@ -1,3 +1,10 @@
-class StatTracker 
+class StatTracker
   def initialize; end
-end
\ No newline at end of file
+
+  def self.from_csv(locations)
+    game_path = locations[:games]
+    teams_path = locations[:teams]
+    game_teams_path = locations[:game_teams]
```

Comment:
Cool. We can add the create_xxx methods once they're created in their respective classes.